beef/merkletree.go

```diff
@@ -0,0 +1,42 @@
+package beef
+
+import (
+	"encoding/hex"
+
+	crypto "github.com/bitcoin-sv/go-sdk/primitives/hash"
+	"github.com/bitcoin-sv/go-sdk/util"
+)
+
+// INFO: This function is moved to go-paymail from go-bt
+// try to use go-sdk implementation when available
+
+// merkleTreeParentStr returns the Merkle Tree parent of two Merkle
+func merkleTreeParentStr(leftNode, rightNode string) (string, error) {
```

Comment:
If the functions are really moved to the `go-sdk`,
maybe we could mark these as `DEPRECATED: ...`

Reply:
@chris-4chain this functions was used by our bump implementation in: `beef/bump.go`
It was adopted from go-bc library but we don't have any equivalent in go-sdk.
Then I moved the implementation from go-bc to get rid of this library completely.

But for your comment - I tried to add proper description to his file and I moved direct comments from original implementation.